daphne_worker/src/durable/aggregate_store.rs

```diff
@@ -138,3 +136,24 @@ impl DurableObject for AggregateStore {
         }
     }
 }
+
+impl DapDurableObject for AggregateStore {
+    fn state(&self) -> &State {
+        &self.state
+    }
+
+    fn deployment(&self) -> crate::config::DaphneWorkerDeployment {
```

Comment:
nit: It would be slightly more general to return a reference to `self..config`, i.e., change the signature here to `config(&self) -> &Confg` (I forget the name of the actual type at the meoment)